instrumentation/vertx-reactive-3.5/javaagent/build.gradle.kts

```diff
@@ -24,23 +34,17 @@ dependencies {
   testInstrumentation(project(":instrumentation:vertx-http-client:vertx-http-client-4.0:javaagent"))
   testInstrumentation(project(":instrumentation:vertx-web-3.0:javaagent"))
 
-  testLibrary("io.vertx:vertx-web-client:${vertxVersion}")
-  testLibrary("io.vertx:vertx-jdbc-client:${vertxVersion}")
-  testLibrary("io.vertx:vertx-circuit-breaker:${vertxVersion}")
   testImplementation("org.hsqldb:hsqldb:2.3.4")
 
-  // Vert.x 4.0 is incompatible with our tests.
-  // 3.9.7 Requires Netty 4.1.60, no other version works with it.
-  latestDepTestLibrary(enforcedPlatform("io.netty:netty-bom:4.1.60.Final"))
-  latestDepTestLibrary("io.vertx:vertx-web:3.+")
-  latestDepTestLibrary("io.vertx:vertx-web-client:3.+")
-  latestDepTestLibrary("io.vertx:vertx-jdbc-client:3.+")
-  latestDepTestLibrary("io.vertx:vertx-circuit-breaker:3.+")
-  latestDepTestLibrary("io.vertx:vertx-rx-java2:3.+")
-}
+  add("version35TestImplementation", "io.vertx:vertx-web:3.5.0")
+  add("version35TestImplementation", "io.vertx:vertx-rx-java2:3.5.0")
+  add("version35TestImplementation", "io.vertx:vertx-web-client:3.5.0")
+  add("version35TestImplementation", "io.vertx:vertx-jdbc-client:3.5.0")
+  add("version35TestImplementation", "io.vertx:vertx-circuit-breaker:3.5.0")
 
-tasks {
-  named<Test>("test") {
-    systemProperty("testLatestDeps", findProperty("testLatestDeps"))
-  }
+  add("latestDepTestImplementation", "io.vertx:vertx-web:4.+")
```

Comment:
Does this separation mean that we actually have to extract the support for vert.x 4+ into a separate instrumentation module?

Reply:
As instrumentation didn't change 3.5 and 4.x can be in the same module, it is only the tests that have changed.